packages/driver/src/util/queue.ts

```diff
@@ -6,111 +6,102 @@ interface QueueRunProps {
   onFinish: () => void
 }
 
-export default {
-  create: <T>(queueables: T[] = []) => {
-    let stopped = false
+export class Queue<T> {
```

Comment:
Can we continue to have this be the default export?

Reply:
There is no written or public guideline for Cypress project. But `default` is usually removed, because it's a translation of `modules.export = {}`. 

Can I ask you why we should use `default` for this?

Reply:
It was previously the default export, so I was just curious why it changed. For files that have a single export that is sufficiently described by the file name, I have preferred a default export over an individual one.

But without an active lint rule or other guidelines to enforce it, this is fine if it's consistent with other files in the package.